pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -332,7 +332,7 @@ private void addProxyDataToDto(UnparsedESParticipantDto esDto, ElasticSearch pro
 
     private void mapTissueToProperOncHistoryDetail(List<OncHistoryDetail> oncHistoryDetails, List<Tissue> tissues) {
         for (Tissue tissue : tissues) {
-            long oncHistoryDetailId = tissue.getOncHistoryDetailId();
+            int oncHistoryDetailId = tissue.getOncHistoryDetailId();
```

Comment:
new change

Reply:
This will not cause a problem but good to make it correct anyway.